cmd/api/main.go

```diff
@@ -9,6 +9,13 @@ import (
 	"stonehenge/app/gateway/api"
 )
 
+// @title Stonehenge API
+// @version 1.0
+// @contact.name Igor Reis (@eakeur)
+// @contact.email igor.reisleandro@gmail.com
+// @securityDefinitions.apikey AuthKey
+// @in header
+// @name Authorization
```

Comment:
❓ **Uma dúvida**: Não seria interessante incluir as` tags` e `descriptions tags` aqui?

```suggestion
// @tag.name Tranfers
// @tag.description endpoints to manage transfers between accounts
```